src/components/customComponents/HeaderNavBar/index.js

```diff
@@ -0,0 +1,41 @@
+export const HeaderNavBar = () => {
+    return <><header>
+    <div className="logo-container">
+        <a href="./index.html">
+            <h1>Artly<span className="logo-text">Store</span></h1>
```

Comment:
change a href to "Link to" in react router dom

Reply:
Yes done.